frame/clearing-house/src/lib.rs

```diff
@@ -52,6 +56,16 @@ pub mod pallet {
 		/// The virtual AMM ID type for this pallet. `pallet-virtual-amm` should implement a trait
 		/// VAMM with an associated type 'VAMMId' compatible with this one.
 		type VAMMId: FullCodec + MaxEncodedLen + TypeInfo;
+		/// Pallet implementation of asset transfers.
+		type Assets: Transfer<
+			Self::AccountId,
+			Balance = Self::Balance,
+			AssetId = Self::MayBeAssetId,
+		>;
```

Comment:
This trait bound is confusing by I've seen it implemented like this elsewhere. `Transfer` itself is actually bounded by the `Inspect` trait from the same crate, but the compiler doesn't complain that I don't make that explicit here